packages/api/src/utils/db-types.d.ts

```diff
@@ -18,6 +18,7 @@ export interface paths {
         query: {
           user_id?: parameters['rowFilter.admin_search.user_id']
           email?: parameters['rowFilter.admin_search.email']
+          github_id?: parameters['rowFilter.admin_search.github_id']
```

Comment:
This PR didn't change this field, but I had to re-generate this massive db-types file as I've added a new `ElasticIpfs` option to the ClusterService enum...